infra/terraform/test-org/org/locals.tf

```diff
@@ -96,5 +96,6 @@ locals {
     "cloud-run",
     "network-forensics",
     "blueprints", # Not module
+    "policy-blueprints",
```

Comment:
Lets also add to https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/1c8897fbd42ceba2a24c6f3178488c568e83baeb/infra/terraform/test-org/ci-triggers/locals.tf#L18 to prevent CB triggers from being created.